NEWS.md

```diff
@@ -1,8 +1,12 @@
 # testthat 0.9.1.9000
 
+* test (test_dir, test_file, test_package, test_check) functions now return a
```

Comment:
I think `test_check()` has a different interface, since it's only ever used in R CMD check.